content/chronograf/v1.7/guides/dashboard-template-variables.md

```diff
@@ -10,24 +10,33 @@ menu:
     parent: Guides
 ---
 
-Chronograf's dashboard template variables allow you to alter specific components of cells' queries
-without having to edit the queries, making it easy to interact with your dashboard cells and explore your data.
+Chronograf's dashboard template variables let you update cell queries without editing queries, making it easy to interact with your dashboard cells and explore your data.
 
 ## Using template variables
-Template variables are used in cell queries and titles when creating Chronograf dashboards.
-Within the query, template variables are referenced by surrounding the variable name with colons (`:`).
 
-```sql
-SELECT :variable_name: FROM "telegraf"."autogen".:measurement: WHERE time < :dashboardTime:
-```
+Be careful when quoting template variables in your queries. When creating a custom meta query, template values are not automatically quoted. Quote template variables as follows:
```

Comment:
I think i might try to describe it like this (and this might go into a separate section about quoting):

For **predefined meta queries** such as "Field Keys" and "Tag Values", Chronograf will determine the appropriate quoting for you, so you do NOT need to add quotes (single or double) to your queries:

```
example goes here
```

For **custom queries, CSV, or map queries** we leave it up to the user to decide when to quote the values in the query in accordance with standard InfluxQL syntax. For numerical values, you do not need to quote them anywhere.

For string values, you can choose to quote the values in the variable definition. For example, if you define a custom CSV variable named `host`, and define it with the following values:

```
'host1','host2','host3'
```

Then you would use them as is in the query the same way you would use the predefined queries, without quoting:

```sql
SELECT mean("usage_user") AS "mean_usage_user" FROM "telegraf"."autogen"."cpu" 
  WHERE "host" = :host: and time > :dashboardTime
```

If you prefer, you can leave the values in the template variable as strings without quotes and quote them when used in the query. In this case, the `host` variable would be defined without quotes:

```
host1,host2,host3
```

and used in the query like this using single quotes for the value:

```sql
SELECT mean("usage_user") AS "mean_usage_user" FROM "telegraf"."autogen"."cpu" 
  WHERE "host" = ':host:' and time > :dashboardTime
```

When using custom meta queries that return strings, you will almost always need to quote the values when using them in a dashboard query, as the results from InfluxQL come back without quotes.

Being able to control the quoting in the query can be useful if you are using template variable strings in regular expression syntax, where quotes would cause query syntax errors.

Reply:
thank you, @russorat!